reakit/reakit
# feat: Add `Role` component

This PR adds a new `Role` component that will eventually replace `Box` (but not yet). `Box` is commonly related to UI libraries that have styles, which creates confusion on people who expect it to accept theming props.

Also, this better matches the most abstract "role" on the WAI-ARIA spec: https://www.w3.org/TR/wai-aria-1.1/#roletype

**How to test?**

Check https://deploy-preview-728--reakit.netlify.app/ and make sure nothing is broken.

**Does this PR introduce breaking changes?**

Not really! But people who are using the experimental system feature might see some wrong behaviors if they're hooking into the Box component (by passing `useBoxProps` to `unstable_system` prop on `Provider`).
